android-sdk/tasks/main.yml

```diff
@@ -50,7 +50,7 @@
 
 -
   name: "Get the Android SDK pod name"
-  shell: "oc get pods --namespace={{ project_name }} | grep android-sdk | awk '{print$1}'"
+  shell: "oc get pods --namespace={{ project_name }} | grep android-sdk | grep -v \"-deploy\" | awk '{print$1}'"
```

Comment:
I think we can just use `grep -v deploy` here. It will still remove the deploy pod and it won't need the additional quotations or escaping the quotations.